MAUI/Circular-Charts/RadialBarChart.md

```diff
@@ -0,0 +1,433 @@
+
+### StartCurve
+
+{% tabs %}
+
+{% highlight xaml %}
+
+    <chart:SfCircularChart>
+    <chart:RadialBarSeries ItemsSource="{Binding Data}" 
+                        XBindingPath="Product" 
+                        YBindingPath="SalesRate" 
+                        CapStyle = "StartCurve"/>
+    </chart:SfCircularChart>
+
+{% endhighlight %}
+
+{% highlight c# %}
+
+    SfCircularChart chart = new SfCircularChart();
+
+        RadialBarSeries series = new RadialBarSeries();
+        series.XBindingPath = "Product";
+        series.YBindingPath = "SalesRate";
+        series.CapStyle = CapStyle.StartCurve;
+
+        chart.Series.Add(series);
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![Radial bar chart start curve cap style in MAUI Chart](Chart-Types_images/maui_startcurve.png)
+
+### EndCurve
+
+{% tabs %}
+
+{% highlight xaml %}
+
+    <chart:SfCircularChart>
+    <chart:RadialBarSeries ItemsSource="{Binding Data}" 
+                        XBindingPath="Product" 
+                        YBindingPath="SalesRate" 
+                        CapStyle = "EndCurve"/>
+    </chart:SfCircularChart>
+
+{% endhighlight %}
+
+{% highlight c# %}
+
+    SfCircularChart chart = new SfCircularChart();
+
+        RadialBarSeries series = new RadialBarSeries();
+        series.XBindingPath = "Product";
+        series.YBindingPath = "SalesRate";
+        series.CapStyle = CapStyle.EndCurve;
+
+        chart.Series.Add(series);
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![Radial bar chart end curve cap style in MAUI Chart](Chart-Types_images/maui_endcurve.png)
+
+## Segment spacing
+
+The [GapRatio]() property of the radial bar series is used to define the spacing between each segments. The default value of this property is `0.2`.
+
+{% tabs %}
+
+{% highlight xaml %}
+
+    <chart:SfCircularChart>
+    <chart:RadialBarSeries ItemsSource="{Binding Data}" 
+                        XBindingPath="Product" 
+                        YBindingPath="SalesRate" 
+                        InnerRadius = "0.2"
+                        GapRatio = "0.4"/>
+    </chart:SfCircularChart>
+
+{% endhighlight %}
+
+{% highlight c# %}
+
+    SfCircularChart chart = new SfCircularChart();
+
+        RadialBarSeries series = new RadialBarSeries();
+        series.XBindingPath = "Product";
+        series.YBindingPath = "SalesRate";
+        series.InnerRadius = 0.2;
+        series.GapRatio = 0.4;
+
+        chart.Series.Add(series);
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![Radial bar chart segment spacing in MAUI Chart](Chart-Types_images/maui_gapratio.png)
+
+## Track customization
+
+You can use the following properties to customize the appearance of the circular bar track.
+
+ * [TrackStroke]() - To customize the circular bar border color.
+
+  * [TrackStrokeWidth]() - To customize the border width of the circular bar.
+
+  * [TrackFill]() - To customize the circular bar area which behind the radial bar segments.
+
+### TrackStrokeWidth
+
```

Comment:
Remove this section.

Reply:
I have removed the track customisation sub headings